gameplay/src/PlatformLinux.cpp

```diff
@@ -1333,6 +1334,13 @@ int Platform::enterMessagePump()
             _game->frame();
         }
 
+        if (!visible && __vsync) {
+        	// glXSwapBuffers may not block on vsync properly if the window is obscured/offdesktop --
+        	// if vsync enabled, do an artificial slowdown - 15ms is largely arbitrary
+        	// if the we have disabled vsync, we don't care, run full speed anyway.
+        	usleep(15000.0);
```

Comment:
The sleep doesn't happen in normal operation -- It only happens when the window is offscreen (minimized, or on a different desktop, etc).  In that case glXSwapBuffers is essentially a noop and the game runs away with a cpu (pretty annoying to have an app grabbing 100% of a cpu only when minimized...)  This just supplies a reason to give up processor time.  The vsync qualifier is just there because if vsync is off, it's already displaying frames as quickly as it possibly can anyway.